src/patternfly/components/DataList/data-list.scss

```diff
@@ -213,32 +217,30 @@ $pf-data-list--breakpoint--mobile: 380px;
 
 // Expandable content
 .pf-c-data-list__expandable-content {
+  position: relative;
   flex: 1 0 100%;
   align-self: flex-start;
   order: 2;
-  max-height: 0;
-
-  // reset li padding on expandable content
-  padding-right: var(--pf-c-data-list__expandable-content--PaddingRight);
-  padding-left: var(--pf-c-data-list__expandable-content--PaddingLeft);
+  max-height: var(--pf-c-data-list__expandable-content--MaxHeight);
 
   // offset for li padding for border to extend to li edges
   margin-right: var(--pf-c-data-list__expandable-content--MarginRight);
   margin-left: var(--pf-c-data-list__expandable-content--MarginLeft);
-  overflow: hidden;
+  overflow-y: auto;
```

Comment:
I was curious why you changed the `max-height` since this created an effect of the content scrolling down as the menu opened when you toggle `.pf-m-expanded` on the list item. That said, the react implementation doesn't surface any sort of animation and the `@keyframes` animation makes the overflow weird. I think as part of this PR we can remove the `opacity`, `transition`, `animation` and the `pf-c-data-list--expandable-content--in` `@keyframes` animation from this.

Reply:
That was actually unintentional. I didn't realize that was for animation :(  I agree with removing this animation though, as we have removed it in almost (I think) every other component.